modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/src/main/resources/META-INF/resources/js/guides/GeneralGuide.js

```diff
@@ -29,6 +29,95 @@ const BORDERS = [
 	'rounded-pill',
 ];
 
+const BORDER_PRIMARY = [
```

Comment:
Can you move all of this to the colors tab instead of the general tab. Make sure you are following the same pattern as what is set for the background colors.

Reply:
Fixed